bigchaindb/README.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+A high-level description of the files and subdirectories of BigchainDB. Heavily used external dependencies include [`multipipes`](https://github.com/bigchaindb/multipipes) and [`bigchaindb-common`](https://github.com/bigchaindb/bigchaindb-common).
+
+## Files
+
+### [`core.py`](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/core.py)
```

Comment:
we can use [github relative linking mechanism](https://help.github.com/articles/relative-links-in-readmes/) to ensure that the file being linked to is of the same branch as the README itself. For the case above:

```
[`core.py`](./core.py)
```